base/variables.scss

```diff
@@ -25,11 +29,16 @@ $header-line-height: 1.25;
 $base-border-radius: 3px;
 $base-spacing: $base-line-height * 1em;
 $base-z-index: 0;
+.clear-margin { margin: 0; }
+.spacebelow-x-2 { margin-bottom: $base-font-size/2 !important; }
```

Comment:
Like the bot says, don't use !important. Not part of variables or mixins. What're you trying to accomplish? Set them on the individual items within the app that the styles are being consumed by.